### PR TITLE
CPS-8-get-tier-config

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
Move get_tier_config to TierConfig class as `get` classmethod.

### CODE REVIEW

File: connect/models/tier_config.py [hunk 1]
```diff
@@ -95,6 +95,36 @@ class TierConfig(BaseModel):
...
+        :param Config config: Config to use, or ``None`` to use environment config (default).
+        :return: The requested Tier Config, or ``None`` if it was not found.
+        :rtype: Optional[TierConfig]
+        """
+        from connect.resources.base import ApiClient
+
+        response, _ = ApiClient(config, base_path='tier/config-requests').get(
+            params={
+                'status': 'approved',
+                'configuration__product__id': product_id,
```
Comment (vgrebenschikov) [inline connect/models/tier_config.py]: Frankly speaking double-underscore notation is unofficial and will be not supported in future 
right notation is dots: configuration.product.id

Comment (JaviCerveraIngram) [reply connect/models/tier_config.py]: OK, I'll change it. Apart from these, FulfillmentAutomation passes the list of products in the 'asset.product.id__in' query param. I assume it should be 'asset.product.id.in' too, right?

Comment (marcserrat) [reply connect/models/tier_config.py]: not for the __in

Comment (JaviCerveraIngram) [reply connect/models/tier_config.py]: Ok, fixed.

**Pull request merged**